integrations-and-supported-tools/pytorch-lightning/scripts/Neptune_Pytorch_Lightning_more_options.py

```diff
@@ -185,9 +186,7 @@ def log_confusion_matrix(lit_model, data_module):
 
     fig, ax = plt.subplots(figsize=(16, 12))
     plot_confusion_matrix(y_true, y_pred, ax=ax)
-    neptune_logger.experiment["confusion_matrix"].upload(
-        neptune.types.File.as_image(fig)
-    )
+    neptune_logger.experiment["confusion_matrix"].upload(File.as_image(fig))
```

Comment:
Is as_image needed here? I guess by default it would be a static image if we use upload(fig)?

Reply:
Updated.
Tests are passing (except timeout and version incompatibility issues), so code should be runnable :)